front/lib/actions/mcp.ts

```diff
@@ -152,12 +153,13 @@ export type MCPToolConfigurationType =
   | ServerSideMCPToolConfigurationType
   | ClientSideMCPToolConfigurationType;
 
-type MCPApproveExecutionEvent = {
+export type MCPApproveExecutionEvent = {
   type: "tool_approve_execution";
   created: number;
   configurationId: string;
   messageId: string;
-  action: MCPActionType;
+  conversationId: string;
```

Comment:
This is more for bubbling up tool approve notification but it's easier to have this already so I can reuse this type